src/main/java/com/crowdin/cli/properties/ParamsWithFiles.java

```diff
@@ -22,6 +22,9 @@ public class ParamsWithFiles extends ProjectParams {
     @CommandLine.Option(names = {"--preserve-hierarchy"}, negatable = true, paramLabel = "...", descriptionKey = "params.preserve-hierarchy")
     private Boolean preserveHierarchy;
 
+    @CommandLine.Option(names = {"--multilingual"}, negatable = true, paramLabel = "...", descriptionKey = "params.multilingual")
+    private Boolean multilingual;
```

Comment:
Can we just allow this parameter in the configuration file and not through the command options? This is a fairly rare use case, and I don't currently see a reason to make the new parameter available here.